src/syslog_mcp/services/alert_monitor.py

```
@@ -0,0 +1,357 @@
+            async with AlertMonitor() as monitor:
+                await monitor.check_all_alerts()
+        except Exception as e:
+            logger.error(f"Alert monitor error: {e}")
+        
+        # Wait for next check
+        await asyncio.sleep(interval_minutes * 60)
+
+
+if __name__ == "__main__":
+    # For testing - run a single alert check
+    asyncio.run(check_alerts_once())
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)